### PR TITLE
fix(@angular/cli): support running a single migration from a package

### PR DESCRIPTION
**Pull request by alan-agius4**
check individual commits

### CODE REVIEW

**Comment by alan-agius4** [comment]
While we can add a schema such as 
```
"migrateOnly": {
  "description": "Only perform a migration, does not update the installed version.",
  "oneOf": [
    {
      "type": "string",
      "description": "The name of the migration to run."
    },
    {
      "type": "boolean"
    }
  ]
},
```

to allow the `migrateOnly` to accept a migration name, this will require some changes in the parser are currently we don't set the values for complex types.

File: packages/angular/cli/commands/update-impl.ts [hunk 1]
```diff
@@ -266,6 +284,10 @@ export class UpdateCommand extends Command<UpdateCommandSchema> {
     } else if (!options.migrateOnly && (options.from || options.to)) {
       this.logger.error('Can only use "from" or "to" options with "migrate-only" option.');
 
+      return 1;
+    } else if (!options.migrateOnly && options.migrationName) {
+      this.logger.error('Can only use "migration-name" option with "migrate-only" option.');
+
       return 1;
     }
```
[the remark this answers is not in the document]
Comment (alan-agius4) [reply packages/angular/cli/commands/update-impl.ts]: Done, I did this as a warning, similar to when using the `--next` option

File: packages/angular/cli/commands/update.json [hunk 2]
```diff
@@ -44,7 +44,19 @@
...
+            {
+              "type": "boolean"
+            },
+            {
+              "type": "string",
+              "description": "The name of the migration to run."
+            }
+          ]
+        },
+        "migrationName": {
```
Comment (clydin) [inline packages/angular/cli/commands/update.json]: Leftover from previous version?

**Comment by dgp1130** [comment]
Merged to `9.0.x` patch branch: https://github.com/angular/angular-cli/commits/9.0.x.

**Pull request merged**